fixtures/input.js

```diff
@@ -942,5 +942,23 @@ module.exports = {
         code: 'isPlainObject([])'
       }
     ]
+  },
+  areEqualArrays: {
+    examples:
+    [
+      {
+        input: [[], []],
```

Comment:
This file is responsible for documentation creation. 
You can see how it looks at https://buxlabs.pl/en/tools/js/pure-conditions/repl 

So I think that input should contains better example that user which read documentations can easily understand what the methods do.